collect_app/src/main/java/org/odk/collect/android/widgets/WidgetFactory.java

```diff
@@ -206,6 +206,8 @@ public static QuestionWidget createWidgetFromPrompt(FormEntryPrompt fep, Context
                     questionWidget = new ListMultiWidget(context, fep, true);
                 } else if (appearance.startsWith("label")) {
                     questionWidget = new LabelWidget(context, fep);
+                } else if (appearance.contains("search") || appearance.contains("autocomplete")) {
```

Comment:
This should ONLY use `autocomplete` for now and not `search` to maintain the `search()` appearance function until we deprecate it.